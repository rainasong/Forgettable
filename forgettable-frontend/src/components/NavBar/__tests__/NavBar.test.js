import ShallowRenderer from 'react-test-renderer/shallow';
import NavBar from '../NavBar';
import React from 'react';
import {BrowserRouter} from 'react-router-dom';


it('renders NavBar UI with correct hierarchy', () => {
  const renderer = new ShallowRenderer();
  renderer.render(<BrowserRouter><NavBar /></BrowserRouter>);
  const result = renderer.getRenderOutput();

  expect(result).toMatchSnapshot();
});


