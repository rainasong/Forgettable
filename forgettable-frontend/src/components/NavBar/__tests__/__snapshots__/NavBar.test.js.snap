// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders NavBar UI with correct hierarchy 1`] = `
<Router
  location={
    Object {
      "hash": "",
      "key": "default",
      "pathname": "/",
      "search": "",
      "state": null,
    }
  }
  navigationType="POP"
  navigator={
    Object {
      "action": "POP",
      "back": [Function],
      "block": [Function],
      "createHref": [Function],
      "forward": [Function],
      "go": [Function],
      "listen": [Function],
      "location": Object {
        "hash": "",
        "key": "default",
        "pathname": "/",
        "search": "",
        "state": null,
      },
      "push": [Function],
      "replace": [Function],
    }
  }
>
  <NavBar />
</Router>
`;
